### PR TITLE
[CM-1117] Added platform flag for User

### PR DESCRIPTION
**Pull request by pahadi777**
Added platform flag in the Android SDK

### CODE REVIEW

File: kommunicate/src/main/java/com/applozic/mobicomkit/api/account/user/User.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 public class User extends JsonMarker {
 
     private static final String DEFAULT_USER_ID_REGEX = "^[a-zA-Z0-9_+#@.?|=;-]+$";
+    private static int platformFlag = 1;
```
Comment (Sathyan-KM) [inline kommunicate/src/main/java/com/applozic/mobicomkit/api/account/user/User.java:20]: Use `platform`

Comment (Sathyan-KM) [reply kommunicate/src/main/java/com/applozic/mobicomkit/api/account/user/User.java:20]: create enum for other platform values

Comment (pahadi777) [reply kommunicate/src/main/java/com/applozic/mobicomkit/api/account/user/User.java]: Changed as suggested

**Review by Sathyan-KM — APPROVED** [review]
Add description to the PR. Add changelog

**Pull request merged**